app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -0,0 +1,342 @@
+  ```
+  $ sudo amazon-linux-extras install postgresql9.6
+  ```
+  or
+  ```
+  $ sudo amazon-linux-extras install posgresql10
+  ```
+  or
+  ```
+  $ sudo amazon-linux-extras install postgresql11
+  ```
+
+2. Install the PostgreSQL Server
+  ```
+  $ sudo yum install postgresql postgresql-server
+  ```
+
+3. Enable the PostgreSQL service
+  ```
+  $ sudo systemctl enable postgresql.service
+  ```
+
+4. Initialize the database
+  ```
+  $ sudo /usr/bin/postgresql-setup --initdb
+  ```
+
+5. Start PostgreSQL
+  ```
+  $ sudo systemctl start postgresql
+  ```
+
+**ii. Create a Kong user and database**
+
+The user and database can easily be created via the command line by using PostgreSQL’s command line tool `psql`
+
+1. Change to the `postgres` user
+  ```
+  $ sudo -i -u postgres
+  ```
+
+2. Launch the PostgreSQL command line tool `psql`
+  ```
+  $ psql
+  ```
+
+3. Create the `kong` user and database
+  ```
+  $ psql> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
+  ```
+  > ⚠️Note: Make sure the username and password for the Kong Database are kept safe. We have used a simple example for illustration purposes only.
+
+4. Exit `psql`
+  ```
+  $ psql> \q
+  ```
+
+5. Return to the terminal
+  ```
+  $ exit
+  ```
+
+**iii. Connect Kong to the PostgreSQL database**
+
+Postgres uses `ident` authentication by default, to allow the `kong` user to communicate with the database locally we must change the authentication method to `md5` by modifying the Postgres configuration file. 
+
+
+1. Open the configuration file:
+  ```
+  $ sudo vi /var/lib/pgsql/data/pg_hba.conf
+  ```
+
+2. Change the `IPV4` local connection method to `md5`
+  ```
+    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
+    | # IPv4 local connections:                      |
+    | host | all      | all  | 127.0.0.1/32 |  md5   |
+    | # IPv6 local connections:                      |
+    | host | all      | all  | ::1/128      | ident  |
+  ```
+
+3. Save and exit the file, then restart Postgres
+  ```
+  $ sudo systemctl restart postgresql
+  ```
+
+Now that the database is set up, Kong must be given credentials to access the database. This is done by adding the Postgres user and password information to Kong’s configuration file.
+
+
+1. Make a copy of Kong’s configuration file
+  ```
+  sudo cp /etc/kong/kong.conf.default kong.conf
+  ```
+
+2. Open the new configuration file
+  ```
+  sudo vi /etc/kong/kong.conf
+  ```
+
+3. Search for `pg_password` under `DATASTORE,` uncomment `pg_password` and add the value. 
+  ```
+  #pg_user = kong
+  pg_password = kong
+  #pg_datastore = kong
+  ```
+  > Note: If you used different values for the user and database name, uncomment the pg_user and pg_datastore variables and add the username and datastore values
+
+
+## Step 4. Customize Your Configuration
+Before starting Kong, you can further modify Kong’s configuration file to enable a host of different features:
+
+
+### Enable RBAC
```

Comment:
Reviewed this today with a couple folks,  we should remove the enable rbac section because it causes more trouble than its worth for someone looking to do a quick install of Kong. lets remove these instructions from here and move them to a section that is more appropriate (i.e. setting up workspaces and teams) later on in the deployment guide.